### PR TITLE
Require raven param

### PR DESCRIPTION
**Pull request by benrudolph**
@wpride this should fix up sentry for the most part. there is still one race condition when the logging of one of the breadcrumbs fails on the initial request. Wasn't able to quickly get to the bottom of it, but it works on subsequent requests oddly. For now going to leave as is because I don't think it's terribly high priority

### CODE REVIEW

File: src/main/java/util/SentryUtils.java [hunk 1]
```diff
@@ -6,31 +6,46 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * Created by benrudolph on 4/27/17.
  */
 public class SentryUtils {
+    /**
+     * We require a Raven instance in these Utils to ensure that the caller has instantiated
+     * the Raven instance. Raven keeps track of the raven instance internally with
+     * Raven.getStoredInstance(). If this call returns null, the Sentry call will fail.
```
Comment (benrudolph) [inline src/main/java/util/SentryUtils.java]: this more or less explains why this is necessary

**Comment by wpride** [comment]
@benrudolph Looks good, guessing you need to mock the Raven in the TestContext

Quoth the raven, Nevermore

**Comment by benrudolph** [comment]
damn you TestContext! thing of evil!

**Comment by wpride** [comment]
Mock.
YEAH
ing.
YEAH
🐦 
YEAH
yeah
YEAH

**Pull request merged**